bindings/go/plugin/client/sdk/plugin_test.go

```diff
@@ -18,13 +19,14 @@ import (
 )
 
 func TestPluginSDK(t *testing.T) {
+	slog.SetLogLoggerLevel(slog.LevelDebug)
 	r := require.New(t)
 
 	output := bytes.NewBuffer(nil)
-	location := "/tmp/test-plugin-plugin.socket"
+	location := "/tmp/test-plugin-flow-plugin.socket"
```

Comment:
suggestion here: use the t.Name() function

Reply:
Will correct in an upcoming one.